### DISCUSSION TITLE
Release Schedule for Version 10.9 and Upcoming Version 11?

### DISCUSSION THREAD

**Discussion by realashleybailey** [jellyfin/jellyfin#9900] [General] [answered]
Although I understand that it may seem unlikely to ask for a general timeline regarding the release date for version 11, I would appreciate any insights or information regarding the potential timeline for the release date of version 10.9. If anyone can provide their knowledge on this matter, it would be greatly appreciated.

**Comment by nielsvanvelzen** [comment, marked as answer]
https://jellyfin.org/docs/general/faq#when-will-release-x-happen-when-will-feature-x-be-available
